tests/multiparser_js_html/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -109,65 +115,45 @@ class MyElement extends LitElement {
 
 customElements.define("my-element", MyElement);
 
-const someHtml1 = html\`
-  <div>hello \${world}</div>
-\`;
-const someHtml2 = /* HTML */ \`
-  <div>hello \${world}</div>
-\`;
+const someHtml1 = html\`<div>hello \${world}</div>\`;
+const someHtml2 = /* HTML */ \`<div>hello \${world}</div>\`;
 
 html\`\`;
 
-html\`
-  <my-element obj=\${obj}></my-element>
-\`;
+html\`<my-element obj=\${obj}></my-element>\`;
 
-html\`
-  <\${Footer}>footer content<//>
-\`;
+html\` <\${Footer}>footer content<//> \`;
 
-html\`
-  <div />
-\`;
+html\` <div /> \`;
+
+html\` <div /> \`;
+
+html\`<span>one</span><span>two</span><span>three</span>\`;
 
 function HelloWorld() {
   return html\`
     <h3>Bar List</h3>
-    \${bars.map(
-      bar => html\`
-        <p>\${bar}</p>
-      \`
-    )}
+    \${bars.map(bar => html\` <p>\${bar}</p> \`)}
   \`;
 }
 
-const trickyParens = html\`
-  <script>
-    f((\${expr}) / 2);
-  </script>
-\`;
-const nestedFun = /* HTML */ \`
-  \${outerExpr(1)}
-  <script>
-    const tpl = html\\\`
-      <div>\\\${innerExpr(1)} \${outerExpr(2)}</div>
-    \\\`;
-  </script>
-\`;
+const trickyParens = html\`<script>
+  f((\${expr}) / 2);
+</script>\`;
+const nestedFun = /* HTML */ \`\${outerExpr(1)}
+<script>
+  const tpl = html\\\`<div>\\\${innerExpr(1)} \${outerExpr(2)}</div>\\\`;
+</script>\`;
```

Comment:
I propose that an extra indentation level be added if there is more than one top-level node in the HTML string.

So we'll get:
```js
const nestedFun = /* HTML */ `${outerExpr(1)}
  <script>
    const tpl = html\`<div>\${innerExpr(1)} ${outerExpr(2)}</div>\`;
  </script>`;
```